apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -161,24 +153,22 @@ export function TokenRow({
 							>
 								Send
 							</TokenRowButton>
-							{isRenderSwapButton && (
```

Comment:
Why remove the check?

Reply:
This will render the `swap` button for all recognized coins. Before we only allow SUI and wUSDC, now we open for all coins
@Jibz-Mysten